OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -304,6 +304,7 @@ public interface IMove
 		bool IsMoving { get; set; }
 		bool IsMovingVertically { get; set; }
 		bool CanEnterTargetNow(Actor self, Target target);
+		bool TurnWhileDisabled(Actor self);
```

Comment:
I filed #13991 as a prerequisite for exposing any new mod-level concepts on `IMove`.

Reply:
OK, I'll update when #13991 is merged in

Reply:
You can rebase now.